docs/semgrep-supply-chain/overview.md

```diff
@@ -54,15 +54,12 @@ your critical/high-severity findings as either reachable or unreachable.
   * If there's a code pattern in the codebase that matches the vulnerability
     definition, the finding is flagged as **reachable**.
       * A finding is **always reachable** if the only way to fix the vulnerability is to upgrade the dependency. Semgrep strongly recommends upgrading the dependencies involved for these findings.
-      * A finding is **conditionally reachable** if Semgrep finds a way to reach it when scanning your code when certain conditions are met.
-  * If you don't use the vulnerable piece of code of the library or package
-  imported, the finding is flagged as **unreachable**.
+      * A finding is **conditionally reachable** if the vulnerability can be exploited when specific conditions are met. Semgrep Supply Chain can determine if the vulnerability is reachable, but such findings require manual review to determine whether those conditions have been met.
+  * If Semgrep Supply Chain determines that you either don't use the vulnerable library package imported or you don't use the vulnerable piece of code of the library or package imported, the finding is flagged as **unreachable**.
   * If Semgrep Supply Chain determines that you use a vulnerable version of a
-  dependency, but Semgrep Supply Chain doesn't have a relevant reachability rule, it flags the finding as **undetermined**; this is most common with
-  vulnerabilities discovered before May 2022 or vulnerabilities with lower
-  severity levels.
+  dependency, but Semgrep Supply Chain doesn't have a relevant reachability rule, it flags the finding as **no reachability analysis**.
```

Comment:
```suggestion
  dependency, but Semgrep Supply Chain doesn't have a relevant reachability rule, it flags the finding as **no reachability analysis**.
```
I didn't get to mention this at the meeting, @misha-semgrep but I definitely thing "no reachability analysis" is a much better term than "undetermined".